src/main.tsx

```diff
@@ -5,11 +5,16 @@ import ReactDOM from 'react-dom/client'
 import { BrowserRouter } from 'react-router-dom'
 
 import { queryClient } from '@/apis/queryClient'
+import { worker } from '@/mocks/worker'
 import { globalStyle } from '@/styles/index.tsx'
 import { theme } from '@/styles/index.tsx'
 
 import App from './App.tsx'
 
+if (process.env.NODE_ENV === 'development') {
+  worker.start()
+}
+
```

Comment:
p5: 이렇게 동적 import를 해도 좋을 것 같아요~!
```suggestion
if (process.env.NODE_ENV === 'development') {
  import('@/mocks/worker').then(({ worker }) => {
    worker.start()
  })
}

```